pkg/sql/schema_changer_test.go

```diff
@@ -3751,12 +3751,19 @@ func TestSchemaChangeRetryError(t *testing.T) {
 		t.Fatal(err)
 	}
 
-	// The timestamp of the transaction is initialized.
 	tx, err := sqlDB.Begin()
 	if err != nil {
 		t.Fatal(err)
 	}
 
+	// The timestamp of the transaction is guaranteed to be fixed after
+	// this statement.
+	if _, err := tx.Exec(`
+		CREATE TABLE t.another (k INT PRIMARY KEY, v INT, pi DECIMAL DEFAULT (DECIMAL '3.14'));
```

Comment:
we could also just observe the current time in tx to get it to be fixed, right? Seems like this works though so don't know if it matters either way.

Reply:
True but this works